common/OpTestHMC.py

```diff
@@ -80,6 +97,41 @@ def check_kernel_signature(self):
             return "signed_kernel"
         return "unsigned_kernel"
 
+    def get_grub_file(self):
+        '''
+        Fetch the grub-file based on the distro
+        '''
+        if self.distro == 'rhel':
```

Comment:
This function code can be reduced:
if self.distro == 'rhel':
    rpm_name = 'grub2-ppc64le'
    filename = 'core.elf'
if self.distro == 'suse':
    rpm_name = 'grub2-powerpc-ieee1275'
    filename = 'grub.elf'

now run the command and check for the file existence.